charts/workbench-operator/Chart.yaml

```diff
@@ -13,9 +13,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.1
+version: 0.2.2
```

Comment:
Attention, same version as in https://github.com/CHORUS-TRE/workbench-operator/pull/32

Reply:
yeah, I'll create only one tag. It's a manual op.